modules/kmm-customizing-upgrades-for-kernel-modules.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/kmm-kernel-module-management.adoc
+
+:_content-type: PROCEDURE
+[id="kmm-customizing-upgrades-for-kernel-modules_{context}"]
+= Customizing upgrades for kernel modules
+
+Use this procedure to upgrade the kernel module while running maintenance operations on the node,
+including rebooting the node, if needed. To minimize the impact on the workload running in the cluster, run the upgrade process
+sequentially, one node at a time. Because it requires knowledge of the workload utilizing the kernel module, this process is
```

Comment:
Not sure this sentence makes sense: "Because it requires knowledge of the workload utilizing the kernel module, this process is managed by the cluster administrator."

I think you can safely remove it.

Reply:
Changed to "This procedure requires knowledge of the workload utilizing the kernel module and must be managed by the cluster administrator."